src/containers/edit-profile/password-security-tab/PasswordSecurityTab.tsx

```diff
@@ -1,34 +1,133 @@
-import { useState } from 'react'
+import { useState, useCallback, FC, ChangeEvent } from 'react'
 import { useTranslation } from 'react-i18next'
 
 import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
 import Divider from '@mui/material/Divider'
 
+import Loader from '~/components/loader/Loader'
 import AppButton from '~/components/app-button/AppButton'
 import AppTextField from '~/components/app-text-field/AppTextField'
 import ConfirmDialog from '~/components/confirm-dialog/ConfirmDialog'
 import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import { snackbarVariants } from '~/constants'
 
 import useForm from '~/hooks/use-form'
+import useAxios from '~/hooks/use-axios'
+import { useAppDispatch } from '~/hooks/use-redux'
+import useInputVisibility from '~/hooks/use-input-visibility'
+
+import { AuthService } from '~/services/auth-service'
+import { openAlert } from '~/redux/features/snackbarSlice'
 
 import { styles } from '~/containers/edit-profile/password-security-tab/PasswordSecurityTab.styles'
-import { confirmPassword, password } from '~/utils/validations/login'
-import { ButtonVariantEnum, InputEnum, SizeEnum, FormValues } from '~/types'
 
-const PasswordSecurityTab = () => {
+import {
+  ButtonVariantEnum,
+  ButtonTypeEnum,
+  InputEnum,
+  SizeEnum,
+  FormValues,
+  UserResponse,
+  ErrorResponse
+} from '~/types'
+import { initialValues, validations } from './PasswordSecurityTab.constants'
```

Comment:
don't forget use `aliases`. a single code import principle

Reply:
Thank you! Fixed.